src/Documentation/doc/advanced/corporation/boost-material.md

```diff
@@ -0,0 +1,187 @@
+# Boost material
+
+## Division production multiplier
+
+Each industry has a different set of boost material's coefficient. For example:
+
+- Agriculture:
+  - AI Cores: 0.3
+  - Hardware: 0.2
+  - Real Estate: 0.72
+  - Robots: 0.3
+- Chemical:
+  - AI Cores: 0.2
+  - Hardware: 0.2
+  - Real Estate: 0.25
+  - Robots: 0.25
+- Tobacco:
+  - AI Cores: 0.15
+  - Hardware: 0.15
+  - Real Estate: 0.15
+  - Robots: 0.25
+
+Division production multiplier is used for calculating [division raw production](./division-raw-production.md) in PRODUCTION state. It's sum of each city's `cityMult`, and `cityMult` is calculated by quantity of boost materials in warehouse and boost material's coefficient of industry.
+
+This multiplier is `this.productionMult` in `Division.ts`.
+
+```typescript
+calculateProductionFactors(): void {
+  let multSum = 0;
+  for (const warehouse of getRecordValues(this.warehouses)) {
+    const materials = warehouse.materials;
+
+    const cityMult =
+      Math.pow(0.002 * materials["Real Estate"].stored + 1, this.realEstateFactor) *
+      Math.pow(0.002 * materials.Hardware.stored + 1, this.hardwareFactor) *
+      Math.pow(0.002 * materials.Robots.stored + 1, this.robotFactor) *
+      Math.pow(0.002 * materials["AI Cores"].stored + 1, this.aiCoreFactor);
+    multSum += Math.pow(cityMult, 0.73);
+  }
+
+  multSum < 1 ? (this.productionMult = 1) : (this.productionMult = multSum);
+}
+```
+
+This is the reason we must expand to all 6 cities. More cities → Higher `this.productionMult` → Higher raw production → More produced materials/products → Higher profit per city → Higher total profit.
+
+Expanding to 6 cities means `this.productionMult` is multiplied by 6, and we have 6 cities, so you can think that production capability of division is multiplied by 36. This is not exactly true because there are more things that affect raw production value of each city, but x36 can be seen as a rough estimate of benefit, especially in early rounds. In those rounds, division production multiplier is the most important thing.
+
+## Optimizer
+
+In order to increase `this.productionMult`, we need to buy boost materials. The problem is how much we should buy each material with a specific constraint of storage space.
+
+Each boost material has a coefficient ("factor" in source code) and a base size (size for 1 unit in storage).
+
+Let's define:
+
+- 4 coefficients: ${c_{1}}$, ${c_{2}}$, ${c_{3}}$, ${c_{4}}$
+- 4 base sizes: ${s_{1}}$, ${s_{2}}$, ${s_{3}}$, ${s_{4}}$
+- Quantities of each boost materials: x, y, z, w
+
+Assume same warehouse setup in all cities: Division production multiplier is:
+
+$$F(x,y,z,w) = \sum_{i = 1}^{6}\left( (1 + 0.002*x)^{c_{1}}*(1 + 0.002*y)^{c_{2}}{*(1 + 0.002*z)}^{c_{3}}{*(1 + 0.002*w)}^{c_{4}} \right)^{0.73}$$
+
+In order to find maximum point of function above, we can find maximum point of this function:
+
+$$F(x,y,z,w) = (1 + 0.002*x)^{c_{1}}*(1 + 0.002*y)^{c_{2}}{*(1 + 0.002*z)}^{c_{3}}{*(1 + 0.002*w)}^{c_{4}}$$
```

Comment:
These math blocks are not rendering corrects in GitHub (they may have an error)

Edit: This is true of *most* of the complicated math expressions in this file.